app/event/[id]/page.tsx

```diff
@@ -0,0 +1,106 @@
+const comments = [
+  {
+    id: 1,
+    author: "alice.bsky.social",
+    content: "looking forward to this! working on a new **react** project",
+  },
+  {
+    id: 2,
+    author: "bob.bsky.social",
+    content: "see you there ☕",
+  },
+];
```

Comment:
I feel like I know you're gonna say no, but eventually we're gonna want this data to come from somewhere.
hopefully we don't miss replacing any placeholder data when that time comes, but I feel like there's no harm in putting something like this outside of the component for now.
that way we can also start thinking about what the data looks like and how it fits into the ui

```suggestion
const event = {
  name: "code & coffee #42",
  date: "tomorrow 9:00am",
  description: "join us for our weekly code & coffee session. bring your laptop, grab a coffee, and work on projects together.\n\nperfect for:\n* networking with local developers\n* getting help with coding problems\n* finding collaborators",
  location: "local cafe\n123 main st\nanytown, usa",
  // joins
  tags: ["javascript", "coffee", "networking"],
  attendees: [
    "alice.bsky.social",
    "bob.bsky.social",
    "charlie.bsky.social"
  ],
  comments: [
    {
      id: 1,
      author: "alice.bsky.social",
      content: "looking forward to this! working on a new **react** project",
    },
    {
      id: 2,
      author: "bob.bsky.social",
      content: "see you there ☕",
    },
  ],
};
```